jenkins-x-boot-lh-ghe.yml

```diff
@@ -61,7 +61,7 @@ pipelineConfig:
 
               - name: build-and-push-image
                 command: /kaniko/executor
-                args: ['--dockerfile=/workspace/source/Dockerfile','--destination=gcr.io/jenkinsxio/jx:${inputs.params.version}','--context=/workspace/source','--cache-repo=jenkins-x-docker-registry.jx.svc.cluster.local:5000/','--cache=true','--cache-dir=/workspace','--skip-tls-verify-registry=jenkins-x-docker-registry.jx.svc.cluster.local:5000']
+                args: ['--dockerfile=/workspace/source/Dockerfile','--destination=gcr.io/jenkinsxio/builder-jx:${inputs.params.version}','--context=/workspace/source','--cache-repo=jenkins-x-docker-registry.jx.svc.cluster.local:5000/','--cache=true','--cache-dir=/workspace','--skip-tls-verify-registry=jenkins-x-docker-registry.jx.svc.cluster.local:5000']
```

Comment:
what does this "builder-" prefix change for this context? wonder whether it would make sense to break the arguments into one argument per line. Might make it easier to spot differences when changes are made.

Reply:
Because for some reason we weren't building the image as `builder-jx` but as `jx` and then never using it. I actually should probably change the `Dockerfile` to use `builder-jx:latest` as the base too. And yeah, good call.